pkg/image/importer/importer.go

```diff
@@ -50,6 +50,9 @@ type ImageStreamImporter struct {
 	limiter   flowcontrol.RateLimiter
 
 	digestToRepositoryCache map[gocontext.Context]map[manifestKey]*api.Image
```

Comment:
Because access to an image may only be allowed given the credentials associated with your context.  So you cannot retry it.  It does not apply to digest to layer size.